pom.xml

```diff
@@ -61,11 +63,17 @@
         <version>${hadoop.version}</version>
         <scope>provided</scope>
       </dependency>
-    <dependency>
-      <groupId>org.hamcrest</groupId>
-      <artifactId>hamcrest-library</artifactId>
-      <version>1.3</version>
-    </dependency>
+      <dependency>
+        <groupId>org.apache.hadoop</groupId>
+        <artifactId>hadoop-aws</artifactId>
+        <version>${hadoop-aws.version}</version>
+        <scope>provided</scope>
```

Comment:
Just to understand, why is this dependency marked as provided here ?

Reply:
the hadoop dependencies are usually on the classpath when you run `hadoop jar ...` This is how circus train runs. So it is up to the application that uses this to make sure the jar is there. If I would make it a compile dependency it would end up in the CT uberjar and it might be a different version  from the one on the classpath etc....

Reply:
Thanks for the explanation.